assets/js/phoenix_live_view.js

```diff
@@ -9,6 +9,7 @@ See the hexdocs at `https://hexdocs.pm/phoenix_live_view` for documentation.
 
 import morphdom from "morphdom"
 
+const PACKAGE_INFO = require('../package.json');
```

Comment:
Does this bring the whole .json file as a dependency? I am also wondering if we should use the package version or use a separate version scheme for the wire protocol. :)

Reply:
Yes, it does bring the whole json file. I think it is fine to use a separate version scheme as long as it is also updated on every release. Don't know how you feel about having to update versioning on an extra file, but would be good so as not to import the package json file. :)

Reply:
This issue is a bit old, but it looks like there are ways webpack can [bake in a version number at compile time](https://github.com/webpack/webpack/issues/237) without importing `package.json`